### PR TITLE
Fix displaying errors in transaction proposal details

### PR DESCRIPTION
**Pull request by JDonadio**
Fixes https://github.com/bitpay/copay/issues/4557

### CODE REVIEW

File: src/js/controllers/modals/txpDetails.js [hunk 1]
```diff
@@ -72,18 +70,30 @@ angular.module('copayApp.controllers').controller('txpDetailsController', functi
...
       walletService.rejectTx(fc, txp, function(err, txpr) {
-        $scope.loading = null;
+        ongoingProcess.set('rejectTx', false);
 
         if (err) {
           $scope.$emit('UpdateTx');
-          $scope.error = bwcError.msg(err, gettextCatalog.getString('Could not reject payment'));
-          $scope.$digest();
+          setError(err, gettextCatalog.getString('Could not reject payment'));
         } else {
```
Comment (isocolsky) [inline src/js/controllers/modals/txpDetails.js]: do not use `else`, instead do ```if (err) return setError(....```

File: src/js/services/onGoingProcess.js [hunk 2]
```diff
@@ -14,6 +14,8 @@ angular.module('copayApp.services').factory('ongoingProcess', function($log, $ti
     'sendingTx': gettext('Sending transaction'),
     'signingTx': gettext('Signing transaction'),
     'broadcastingTx': gettext('Broadcasting transaction'),
+    'rejectTx': gettext('Rejecting payment proposal'),
+    'removeTx': gettext('Deleting Payment proposal'),
```
Comment (isocolsky) [inline src/js/services/onGoingProcess.js]: use lowercase `p`

Comment (JDonadio) [inline src/js/services/onGoingProcess.js]: Good catch!

**Comment by JDonadio** [comment]
Updated. Thanks @isocolsky !

**Comment by isocolsky** [comment]
ACK

**Pull request merged**